Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/OthersContentPage.xaml

```diff
@@ -3,6 +3,7 @@
     <TableView Intent="Form">
         <TableSection Title="Distribute">
             <SwitchCell Text="Distribute Enabled" On="true" x:Name="DistributeEnabledSwitchCell" OnChanged="UpdateDistributeEnabled" />
+            <SwitchCell Text="Distribute uses private track" On="true" x:Name="DistributePublicSwitchCell" OnChanged="UpdateDistributeTrackUpdate" />
```

Comment:
Same feedback as Android, the check boxes are really confusing, we should rather use a dropdown like the start mode.

Reply:
Also it's missing the before start vs now setting.